### PR TITLE
restructuring AIT3 docs + add max gas in CLI

### PR DESCRIPTION
**Pull request by wintertoro**
I want to restructure. the AIT3 docs in this way as I think it would be significantly clearer. I also fixed some small bugs like max-gas missing from CLI instructions. Edited based on greg's comments. 

 There’s a ton of replication between the docs originally

Like initialize staking is spread out over three docs originally. So I put all the stuff to do with CLI, voting, and staking (incl) Petra wallet, in one doc. Then all the stuff to do with joining the network, leaving the network is consolidated in another doc. Its a struggle to jump between the docs in multiple places to finish a sequence of instructions. 

Also the same CLI instructions appeared I think 2-3 times in a few places, so I removed the duplicates. Some links might be broken, but it would be easier to fix those after this PR has been merged

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3707)

### CODE REVIEW

File: terraform/helm/fullnode/values.yaml [hunk 1]
```diff
@@ -39,10 +39,10 @@ image:
 
 resources:
   limits:
-    cpu: 7
+    cpu: 6.5
```
Comment (rustielin) [inline terraform/helm/fullnode/values.yaml:42]: nit: probably keep the CPU limit at 7, and only change the request

Comment (wintertoro) [reply terraform/helm/fullnode/values.yaml]: resolved

**Comment by rajkaramchedu** [comment]
Docs build is failing for this PR. These docs require some knowledge of how Docusaurus works for docs placement. I will not spend time debugging the docs build failure, but will look at the changes proposed here and pick them into separate PR.

**Comment by wintertoro** [comment]
Sounds good!

Jing

**Pull request closed without merging**